ci/redmine_bugzilla.py

```diff
@@ -172,6 +183,9 @@ def main():
                         links_issues_record += 'Bugzilla #%s -> Redmine %s, but Redmine %s does ' \
                                                'not link back\n' % (bug.id, issue.id, issue.id)
 
+    end_time = time.time()
+    time_to_query_all = end_time - start_time
+    print 'Total time to query %d redmine issues: %f' % (redmine_issues_looked_up, time_to_query_all)
```

Comment:
This is a duplicate block, and should probably be wrapped in a function.

In both cases where this value is printed, it would be helpful to see the unit of time measured.